llvm/docs/ReleaseNotes.md

```diff
@@ -130,6 +130,9 @@ Changes to the Debug Info
 Changes to the LLVM tools
 ---------------------------------
 
+- llvm-objcopy now prints the correct file path in the error message when
+the output file specified by --dump-section cannot be opened.
```

Comment:
IIRC the wrapped line should be indented in markdown

If this is going to be merged into release/20.x, then this could be removed to avoid a merge conflict.

Reply:
Thank you, I will remove it from this PR and merge after CI become green, then open pr to put it on release